README.md

```diff
@@ -1,3 +1,7 @@
 # FTB-App
 
-You can find the java backend side of the FTB launcher [here](https://github.com/CreeperHost/modpacklauncher)
\ No newline at end of file
+This is the frontend of the Official Feed The Beast App.
+
+## Backend
+
+You can find the java backend side of the FTB launcher [here](https://github.com/CreeperHost/modpacklauncher)
```

Comment:
Should say App and not launcher?

Reply:
Good point, that text is actually from before this PR so I'll change that now